docs/GroupDefs.hpp

```diff
@@ -485,6 +485,11 @@
  * components.
  */
 
+/*!
+ * \defgroup InitialDataGroup Initial Data
```

Comment:
Do we really need or want another group for this? What's going to go in this group? If the initial data stuff is mostly geared towards evolution, I'd say put the `InitialDataUtilities` namespace in the `EvolutionSystemsGroup` :shrug: It's not clear to me what the right place is, but having a separate thing for just one class seems to me to be the wrong thing to do.